src/react-components/ui-root.js

```diff
@@ -540,6 +532,33 @@ class UIRoot extends Component {
             <a href="https://github.com/mozilla/hubs">Github</a>.
           </div>
         );
+      } else if (this.props.platformUnsupportedReason === "no_data_channels") {
+        // TODO i18n, due to links and markup
+        subtitle = (
+          <div>
+            Your browser does not support{" "}
+            <a href="https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel" rel="noreferrer noopener">
```

Comment:
Point to https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createDataChannel#Browser_compatibility